quarkus/tests/integration/pom.xml

```diff
@@ -62,4 +62,39 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <skipTests>true</skipTests>
```

Comment:
Sorry, I think I misunderstood when you mentioned that to me before. Is it really a good idea to disable tests by default?

If we are avoiding running in unit tests, perhaps we should just add a `skipQuarkusTests` similarly to the `skipTestsuite`. See https://github.com/keycloak/keycloak/blob/6a766884f632858990a05dabc4019ff3f18d972c/pom.xml#L2024.